### PR TITLE
Nerf cloaking device

### PR DESCRIPTION
**Pull request by toilethinges**
Multiply energy by 7.5x, fuel by 1.1x and heat by... some amount, it wasn't there before.

450 energy drain and 210 heat caused, as well as draining ten percent more fuel than before.

Taking suggestions on number changes.

### CODE REVIEW

File: data/pug.txt [hunk 1]
```diff
@@ -461,7 +461,7 @@ ship "Pug Arfecta"
 		"hull energy" 5
 		"heat generation" 10
 		"energy capacity" 10000
-		"cloaking fuel" -.2
+		"cloaking fuel" -.22
 		weapon
```
Comment (KiLEdEnNis) [inline data/pug.txt]: is it me or is this line and the 2 below it missing a tab? (which is irrelevant to the PR but I just noticed it)

**Comment by KiLEdEnNis** [comment]
I think I found a mistake that was already there when I was just looking at your edits

**Comment by KiLEdEnNis** [comment]
seems good

**Comment by Wrzlprnft** [comment]
can you explain a little where the numbers came from initially?

**Comment by zwparchman** [comment]
What is the goal for this nerf? 

Nerfing the numbers will just push it more and more toward being an outfit that is only useful on a heavy warship, which is where it is the least useful.

**Comment by Bladewood** [comment]
Would need about 13 ramscoop power here to indefinitely sustain cloaking (at a distance of 0 units), in compared to about 11 from the previous version.
That is, 3 wanderer ramscoops in the previous version, and 4 in this version.
(all numbers are rounded up)

**Comment by toilethinges** [comment]
I don't really understand what anyone wants, so I'm letting someone more capable open another one.

**Comment by Elyssaen** [comment]
I do think I agree with @zwparchman, so am glad of that. I get that there are balancing challenges with the Cloaking Device, but I don't think changing the energy cost is the answer. The answer, if one is needed (and it might be when higher tiers begin to appear, and "nothing can beat an Arfecta" becomes a problem), is probably more complicated and involves the cloaking not being absolute.

**Comment by KiLEdEnNis** [comment]
having a few fast ships with flamethrowers to keep it permanently overheated can easily beat it, you can pull it off with a few medium/heavy human warships (like a flamethrower Vanguard and a couple of Falcons)

**Comment by Elyssaen** [comment]
Oh, sure. The player can do it: I'm sure most experienced players have captured at least some of the Arfectas, if not all. But you can imagine higher tier combat where it might start to get useful to have the ability for some ships to _see through_ some Cloaking Devices.

**Comment by KiLEdEnNis** [comment]
depends, if there are actually gonna be fleets of cloaking ships, which I doubt

**Comment by koflerdavid** [comment]
A way to nerf it would be to tie energy and fuel requirements to the ship's mass. This should be enough to keep it usable for small ships, while making it difficult for large ones. The Pug with their superior tech can of course still cloak their Arfectas (aren't energy production and shields of Pug ships hardcoded?)

**Comment by Wrzlprnft** [comment]
i personally wouldn't nerf the cloaking at all. it's one of the few things which benefit single ship playstyles as long as as few of them are obtainable as possible

**Comment by KiLEdEnNis** [comment]
plus you need a lot of ramscoops/fuel pods to make good use of it (unless you wanna run outta fuel)

**Pull request closed without merging**